.circleci/config.yml

```diff
@@ -74,6 +74,6 @@ workflows:
           matrix:
             parameters:
               ruby_version:
-                - '2.5'
```

Comment:
I still prefer that dropping the old version is a separate commit 😁

Reply:
Perfect, I will split it